webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,17 @@ function BugForm() {
     }, this));
   };
 
+  // See function autoExpand in issues.js
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand",function(){})
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
+      this.style.overflow = 'hidden';
```

Comment:
If you run eslint on this code, you'll see a few errors that should be fixed up (double quotes, indentation, etc.)

There's more details on the travis results: https://travis-ci.org/webcompat/webcompat.com/builds/216025269